vitest.config.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="vitest" />
```

Comment:
Is this comment needed?

Reply:
its a compiler directive, it a a hint to the TypeScript compiler